tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -10615,6 +10615,41 @@ public function testInferPrivatePropertyTypeFromConstructor(
 		);
 	}
 
+	public function dataInferPrivateTypedPropertyTypeFromConstructor(): array
```

Comment:
The more modern way of testing such stuff is done in NodeScopeResolverTest::testFileAsserts(), please rework your tests based on that.